pwa/src/components/translations/table.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react";
+// import "../../style/table.css";
+
+interface TableProps {
+  columns: Array<Partial<Record<"field" | "headerName" | "renderCell" | "hidden" | "valueFormatter", any>>>;
+  rows: Array<Record<any, any>>;
```

Comment:
Why are we using such complex types (an `Array` that holds a `Partial` that holds a `Record`). Is this really necessary? Also, the use of `any` really does beat the purpose most of the time.

Especially the `rows: Array<Record<any, any>>;` doesn't say too much since we're passing `any` twice.

Reply:
But besides the `any` argument, couldn't we make this a much easier type and achieve the same outcome?